### PR TITLE
Dependencies update 

### PR DESCRIPTION
**Pull request by flampouris**
This PR  1. provides updates on the Readme.md file regarding the required libraries and 2. minor updates of gdas_utility.py to adjust the requirement for AWS credentials at the appropriate level. 

Details in the description of each commit.

### CODE REVIEW

**Comment by SadeghTabas-NOAA** [comment]
@flampouris Thanks for the PR.

**Review by SadeghTabas-NOAA — APPROVED** [review]

File: NCEP/gdas_utility.py [hunk 1]
```diff
@@ -635,11 +635,11 @@ def get_dataarray(self, grbfile, var_name, level_type, desired_level):
         os.environ['AWS_CONFIG_FILE']=custom_config_file
 
     # check environment variables
-    if (download_source == 's3') & (os.environ.get('AWS_SHARED_CREDENTIALS_FILE') is None) | (os.environ.get('AWS_CONFIG_FILE') is None):
-        if os.environ.get('AWS_SHARED_CREDENTIALS_FILE') is None:
-            raise ValueError('Please set up environment varialbes AWS_SHARED_CREDENTIALS_FILE')
-        if os.environ.get('AWS_CONFIG_FILE') is None:
-            raise ValueError('Please set up environment varialbes AWS_CONFIG_FILE')
+        if (download_source == 's3') & (os.environ.get('AWS_SHARED_CREDENTIALS_FILE') is None) | (os.environ.get('AWS_CONFIG_FILE') is None):
```
Comment (junwang-noaa) [inline NCEP/gdas_utility.py:638]: @flampouris May I ask why you need the change here?

Comment (flampouris) [reply NCEP/gdas_utility.py:638]: @junwang-noaa , @SadeghTabas-NOAA covered it. In principle, I was testing with publicly available data, e.g., NOMADS, and the conditional statement required to have AWS credentials.

**Review by flampouris — COMMENTED** [review]
I hope that my comment on the conditional statement answers the question. Please let me know if there is anything else. 

Thank you,
stelios

File: NCEP/gdas_utility.py (hunk 1 above)
Comment (junwang-noaa) [reply NCEP/gdas_utility.py:638]: @flampouris Since Sadegh will make a PR to address the issue, so we don't need the changes here.

**Comment by SadeghTabas-NOAA** [comment]
> I hope that my comment on the conditional statement answers the question. Please let me know if there is anything else.
> 
> Thank you, stelios

@flampouris Thanks Stelios for the clarification, I'll address this in this issue https://github.com/NOAA-EMC/graphcast/issues/40 , so I close this. Thanks

File: NCEP/gdas_utility.py (hunk 1 above)
Comment (SadeghTabas-NOAA) [reply NCEP/gdas_utility.py:638]: @flampouris I also wanted to note that in the next PR we will update S3 credentials for public use, so you can use s3 source option. The updated s3 bucket also provide historical gdas data for graphcast ICs. Thanks

**Pull request closed without merging**